### PR TITLE
Update CHANGELOG for 2.1.1 Release

### PR DESCRIPTION
**Pull request by kealjones-wk**
## Motivation
  
- Needed to update the changelog to include latest stuff before release

## Changes
  
- Updated the changelog

## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

### QA Checklist
- [ ] Tests were updated and provide good coverage of the changeset and other affected code
- [ ] Manual testing was performed if needed
    - [ ] Steps from PR author:
        
    - [ ] Anything falling under manual testing criteria [outlined in CONTRIBUTING.md][contributing-manual-testing]

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:
- [ ] A Frontend Frameworks Design member has reviewed these changes
- [ ] There are no unaddressed comments _- this check can be automated if reviewers use the "Request Changes" feature_
- [ ] _For release PRs -_ Version metadata in Rosie comment is correct

[contributing-review-types]: https://github.com/Workiva/react_testing_library/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/react_testing_library/blob/master/CONTRIBUTING.md#manual-testing-criteria

### CODE REVIEW

**Review by sydneyjodon-wk — APPROVED** [review]
+10

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Comment by kealjones-wk** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,17 @@
 # React Testing Library Changelog
 
+## 2.1.1
+* [#54](https://github.com/Workiva/react_testing_library/pull/54) Update CHANGELOG for 2.1.1 Release
+* [#53](https://github.com/Workiva/react_testing_library/pull/53) Remove a CI check that was no longer working properly
+* [#52](https://github.com/Workiva/react_testing_library/pull/52) Fix missing arguments in formatted output from forwarded console logs
+* [#51](https://github.com/Workiva/react_testing_library/pull/51) Update references to the previous team name with Frontend Frameworks Design
+* [#49](https://github.com/Workiva/react_testing_library/pull/49) Add changelog for 2.1.0 release, add de-facto skynet plan -
```
Comment (greglittlefield-wf) [inline CHANGELOG.md:8]: Thanks for adding this changelog!

#nit, changes like changelog updates and most CI cganges don't impact consumers and IMO make the changelog noisy, so in the future we can omit those

**Pull request merged**